### DISCUSSION TITLE
Difficulty in finding data

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#40] [General]
The lack of a unique platform that allows discover thematic available data on the basis of an Area of Interest, (polygon) defined by the user, difficult the finding of available data in that area/region for average citizens. 

In would be desirable and very usable to have a open spatial inventory, by theme, in which the metadata (spatial metadata) of both INSPIRE and non-INSPIRE data would be grouped in cells, allowing data location in the area established by the user/requestor. This kind of tool would facilitate data access to citizenship.

The image below has been created to illustrate the spatial inventory proposed. 

![image](https://user-images.githubusercontent.com/94920015/199927727-da5da84f-4a13-462f-9119-30a587c520e9.png)

**Comment by naomithiru** [comment]
A difficulty in finding 'national' sewer network datasets has been encountered in the Tracing use-case. Network, in this case is either an edge network, or a node-edge network. In most countries, only discharge points are more easily available. To counter this, our processing functions allow for these scenarios, although having a network is more ideal than working with only the discharge points.
A different strategy to obtain more localized sewer dataset is currently being undertaken.

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.2.5 in D4.3

From a user perspective, the problem is that the user has to query several different portals in order to find relevant datasets. This means that the metadata catalogues cannot be accessed by a single portal. One objective of the national geoportals and the national data portals is to provide such a single access point.

**Reply by AndersOstmanNovogit** [reply]
Moved to D4.3 section 4.1.3
